README.md

```diff
@@ -1239,7 +1239,7 @@ I'm Viacheslav. https://github.com/dviacheslav
 
 
 
-Hello! 
+Hello!
```

Comment:
Hi Dima! 
Your code if good! But you a little bit change previous code in this file (even if it just a `space` or `tab`)
I guess the main reason you did changes like this
![Screenshot](https://iili.io/24Svls.md.png)
is that you use some autoFormat settings in you redactor/IDE. You can try to turn off this behaviour (at least during this task).
P.S. To roll back in repo files history you, for example, can use `git log` and `git checkout commitSHA`